tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -723,13 +723,13 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiHeadroomPoolSize[None]:
   skip:
     reason: "Headroom pool size not supported."
     conditions:
-      - "hwsku not in ['Arista-7060CX-32S-C32', 'Celestica-DX010-C32', 'Arista-7260CX3-D108C8', 'Force10-S6100', 'Arista-7260CX3-Q64', 'Arista-7050CX3-32S-C32'] and asic_type not in ['mellanox']"
+      - "hwsku not in ['Arista-7060CX-32S-C32', 'Celestica-DX010-C32', 'Arista-7260CX3-D108C8', 'Force10-S6100', 'Arista-7260CX3-Q64', 'Arista-7050CX3-32S-C32', 'Nokia-IXR7250E-36x100G', 'Nokia-IXR7250E-36x400G'] and asic_type not in ['mellanox']"
```

Comment:
@vmittal-msft  - this test is passing for us, except for the in_discard stats. Why are we adding 'skip' for this test on Nokia.